[7983] Change apostrophes from straight to curly style (#5043)
DFE-Digital/register-trainee-teachers: docs/support_playbook.md

## Making data changes

> If you’re making a data change, you must include an `audit_comment` so that we can see why we did this. E.g

Reviewer: ```suggestion
If you’re making a data change, you must include an `audit_comment` so that we can see why we did this. For example,
```

Dqt::RetrieveTeacher.call(trainee: t)
```

> If the trainee doesn’t have a TRN yet, call this instead:

Reviewer: ```suggestion
If the trainee does not have a TRN yet, call this instead:
```

```

> All record changes should be sent to DQT unless otherwise specified or impossible (e.g. we can’t send a DOB update). If DQT already has that info (e.g. they’re awarded on DQT, and we’re just awarding on Register) we should not send any information.

Reviewer: ```suggestion
All record changes should be sent to DQT unless otherwise specified or impossible (for example, we cannot send a DOB update). If DQT already has that info (for example, they’re awarded on DQT, and we’re just awarding on Register) we should not send any information.
```

This command will:
1. Open the `trainees.csv` file.
2. Run through each row, using the “id” field to find the corresponding trainee in the database.
3. If found, it will find and discard any other trainee that has the same email address.
4. If not found, it will print a message stating that the trainee was not found.

> 5. Once the task finishes running through all rows in the CSV, it will print "Task completed."

Reviewer: ```suggestion
5. Once the task finishes running through all rows in the CSV, it will print “Task completed.”
```
